### DISCUSSION TITLE
Ordinal Script Encode

### DISCUSSION THREAD

**Discussion by devlancer412** [paulmillr/micro-btc-signer#76] [Q&A]
Could you provide me a example encode code for this script?

```
OP_FALSE
OP_IF
  OP_PUSH "ord"
  OP_PUSH 1
  OP_PUSH "text/plain;charset=utf-8"
  OP_PUSH 2
  OP_PUSH 0xff
  OP_PUSH 0
  OP_PUSH "Hello, world!"
OP_ENDIF
```

I can't find some commands from OP code list

**Comment by paulmillr** [comment]
Someone will need to figure it out and post the answer here.

**Reply by devlancer412** [reply]
Could you please be that 'Someone'?

**Reply by paulmillr** [reply]
No. I have no plans on diving into ordinals right now. Also see other discussions https://github.com/paulmillr/scure-btc-signer/discussions
